README.md

```diff
@@ -1,2 +1,228 @@
-# satellite6-upgrade
-Satellite 6 upgrade related tools
+# Satellite6-upgrade
+
+Satellite6 upgrade is a set of tools to be used for following reasons:
+1. Creating and deleting satellite/capsule instances using RHEVM templates/Openstack images
+2. Upgrade Satellite6, Capsule and clients (RHEVM instances / Users own setup)
+3. Post Upgrade Satellite Entities Verification
+
+## Installation
```

Comment:
Before this Section we may need to add Supported upgrade paths  through satellite6 upgrade for both satellite and capsule